server/server.js

```diff
@@ -114,6 +119,45 @@ app.get(`${apiBaseUrl}/participants`,
     return res.json(result);
   }));
 
+app.get(`${apiBaseUrl}/participant`,
+  keycloak.allowRolesMiddleware('health_authority', 'ministry_of_health', 'employer'),
+  keycloak.getUserInfoMiddleware(),
+  asyncMiddleware(async (req, res) => {
+    await validate(ParticipantQuerySchema, req.query);
+    const user = req.hcapUserInfo;
+    const id = req.query;
+    const result = await getParticipantByID(id);
+    logger.info({
+      action: 'participant_get',
+      performed_by: {
+        username: user.username,
+        id: user.id,
+      },
+      on: {
+        id,
+      },
+    });
+    return res.json(result);
+  }));
+
+// Update participant data
+app.post(`${apiBaseUrl}/participant`,
+  keycloak.allowRolesMiddleware('health_authority', 'ministry_of_health', 'employer'),
+  keycloak.getUserInfoMiddleware(),
+  asyncMiddleware(async (req, res) => {
+    // await validate(ParticipantQuerySchema, req.body);
```

Comment:
should be cleaned up :)

Reply:
Oh! Good reminded that I never implemented validation. I'll get on that.